examples/dynamic-bridge-app/linux/include/Device.h

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/app-common/zap-generated/cluster-enums.h>
+#include <lib/support/Span.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "Clusters.h"
+
+// Device types for dynamic endpoints: TODO Need a generated file from ZAP to define these!
+// (taken from chip-devices.xml)
+#define DEVICE_TYPE_BRIDGED_NODE 0x0013
+// (taken from lo-devices.xml)
+#define DEVICE_TYPE_LO_ON_OFF_LIGHT 0x0100
+// (taken from lo-devices.xml)
+#define DEVICE_TYPE_LO_ON_OFF_LIGHT_SWITCH 0x0103
+// (taken from chip-devices.xml)
+#define DEVICE_TYPE_ROOT_NODE 0x0016
+// (taken from chip-devices.xml)
+#define DEVICE_TYPE_BRIDGE 0x000e
+
+// Device Version for dynamic endpoints:
+#define DEVICE_VERSION_DEFAULT 1
```

Comment:
Please separate out these into their own 'DeviceTypesList.h` or something.

Reply:
Moved the only used ones to cpp